addons/sale/models/sale_order.py

```diff
@@ -218,6 +218,8 @@ def _rec_names_search(self):
     amount_untaxed = fields.Monetary(string="Untaxed Amount", store=True, compute='_compute_amounts', tracking=5)
     amount_tax = fields.Monetary(string="Taxes", store=True, compute='_compute_amounts')
     amount_total = fields.Monetary(string="Total", store=True, compute='_compute_amounts', tracking=4)
+    amount_invoiced = fields.Monetary(string="Already invoiced", store=True, compute='_compute_invoiced_amounts')
+    amount_to_invoice = fields.Monetary(string="Amount to invoice", store=True, compute='_compute_invoiced_amounts')
```

Comment:
Since `amount_to_invoice` is displayed in the tree view, it makes sense to store it.
But why do you need to store `amount_to_invoice`?

Reply:
Just so I can use the same compute method :sweat_smile: 
It looked cleaner to me, but I don't know if it is enough to justify storing a field ? :wink:

Reply:
But isn't `amount_to_invoice = amount_total - amount_invoiced`?
It's a one line compute function, no need to merge both, nor to store both